files/main.css

```diff
@@ -304,12 +304,6 @@ h1 a {
 			margin: 1px 0;
 		}
 
-		#panel #content h2 {
-			margin-top: 32px;
-			border-top: var(--border-style);
-			padding-top: 12px;
-		}
-
 		#panel #content h2:first-child {
```

Comment:
Can you please double check but I think this rule is not required anymore.

Reply:
You are right, docs share the same code 🤔
What do you suggest?

Reply:
> You are right, docs share the same code

The problem is that the examples and docs use the same CSS but diverted in their logic. If  the `#panel #content h2` related styles are removed, the docs will be affected. 

#19943 has the advantage that it solves the issue without affecting the docs. It requires JS though.

In any event, I don't think we want to make a change that affects the docs.

Reply:
Yeah, I think this can be closed and you can go with JS solution so that we don't have to make changes to docs.

Do you think it would be worth it to either
- try and bring back docs & examples to a common code base
- completely separate the CSS ( except common stuff like typography, colors, etc)
?

Reply:
Not sure about this 🤔 .

But thanks for give a pure CSS approach a try 👍 .